docs/api.md

```diff
@@ -412,7 +412,7 @@ defaultLogLevel: 'error'
 Note that this value can be overriden by passing 'logLevel' as a prop when rendering the component.
 
 
-#### autoResize `boolean | Object<String, Boolean>`
+#### autoResize `boolean | Object<String, Boolean | String>`
```

Comment:
Can we change this to `boolean | { height : boolean, width : boolean, element : string }`